src/collections/_documentation/performance-monitoring/performance/event-detail.md

```diff
@@ -63,3 +69,90 @@ Traversing between parent and child transactions is only available on the [Busin
     content=__alert_content
     level="info"
 %}
+
+**Grouping Transactions**
+
+When Sentry captures transactions, they are assigned a transaction name. This name is generally auto-generated by the Sentry SDK based on the framework integrations you are using. If you can't leverage the automatic transaction generation (or want to customize how transaction names are generated) you can use a global event processor that is registered when you initialize the SDK with your configuration.
+
+An example of doing this in a node.js application:
+
+```javascript
+import { addGlobalEventProcessor } from '@sentry/node';
+
+addGlobalEventProcess(event => {
+  // if event is a transaction event
+  if (event.type === "transaction") {
+	event.transaction = sanitizeTransactionName(event.transaction);
+  }
+  return event;
+});
+```
+
+For browser JavaScript applications using the `Tracing` integration, the `beforeNavigate` option  can be used to better group navigation/pageload transactions together based on URL.
+
+```javascript
+import * as Sentry from '@sentry/browser';
+import { Integrations as ApmIntegrations } from '@sentry/apm';
+
+Sentry.init({
+	// ...
+  integrations: [
+    new ApmIntegrations.Tracing({
+      beforeNavigate: (location) => {        
+        // You could use your UI's routing library to find the matching
+		// route template here. We don't have one right now, so do some basic
+		// parameter replacements.
```

Comment:
```suggestion
      beforeNavigate: (location) => {        
        // You could use your UI's routing library to find the matching
        // route template here. We don't have one right now, so do some basic
        // parameter replacements.
```

Looks like you might have mixed tabs/spaces here or the indentation is just inconsistent.